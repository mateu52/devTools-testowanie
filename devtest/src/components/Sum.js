//import React, { useState } from 'react';
//import ErrorBund from "../ErrorBund";




function Sum(x, y){
    const sum = x+y;

    return sum;
  }
export default Sum;