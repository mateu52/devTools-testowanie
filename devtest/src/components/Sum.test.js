import React from "react";
import Sum from "./Sum";
import renderer from 'react-test-renderer';
 //import ShallowRenderer from 'react-test-renderer/shallow';


it('renders ok', () => {
    const tree = renderer
        .create(<Sum />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})


describe('sum = a+b' , () => {
    it('sum is a+b', () => {
        expect(Sum(1,2)).toBe(3);
    });

})
