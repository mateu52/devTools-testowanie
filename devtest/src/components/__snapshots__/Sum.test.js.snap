// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ok 1`] = `"[object Object][object Object]"`;
